src/metpy/calc/thermo.py

```diff
@@ -2750,6 +2750,26 @@ def thickness_hydrostatic(pressure, temperature, mixing_ratio=None,
     `pint.Quantity`
         The thickness of the layer in meters
 
+    Examples
+    --------
+    >>> import metpy.calc as mpcalc
+    >>> from metpy.units import units
+    >>> temperature = [278, 275, 270] * units.kelvin
+    >>> pressure = [950, 925, 900] * units.millibar
+    >>> mpcalc.thickness_hydrostatic(pressure, temperature)
+    <Quantity(434.376889, 'meter')>
+
+    >>> bottom, depth = 950*units.millibar, 25*units.millibar
```

Comment:
Just a minor tweak to fit the rest of the style in this example (and the code base):

```suggestion
    >>> bottom, depth = 950 * units.millibar, 25 * units.millibar
```